1.10/deploying-services/uninstall.md

```diff
@@ -4,7 +4,7 @@ nav_title: Uninstalling
 menu_order: 007
 ---
 
-Services can be uninstalled from either the web interface or the CLI. If a Universe service has any reserved resources, you also need to run the framework cleaner script. The [framework cleaner script](#framework-cleaner) removes the service instance from ZooKeeper, along with any data associated with it.
+Services can be uninstalled from the CLI. If a Universe service has any reserved resources, you also need to run the framework cleaner script. The [framework cleaner script](#framework-cleaner) removes the service instance from ZooKeeper, along with any data associated with it.
```

Comment:
It doesn't have to happen on this PR, but this isn't quite correct. _Some_ services can be uninstalled in the GUI, others can't. And as of 1.10, many services don't need the janitor script.